config.toml

```diff
@@ -138,13 +138,13 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.25"
+latest = "v1.26"
 
-fullversion = "v1.25.0"
-version = "v1.25"
-githubbranch = "main"
-docsbranch = "main"
-deprecated = false
+fullversion = "v1.26.0"
+version = "v1.26"
+githubbranch = "v1.26.0"
+docsbranch = "release-1.26"
+deprecated = true
```

Comment:
remove this line?

Reply:
not sure

Reply:
Rather, the original needs to be retained.

Reply:
@krol3 Please change

githubbranch = "main"
docsbranch = "main"
deprecated = false

Reference: [1.24 PR](https://github.com/kubernetes/website/pull/33412)